alertMaintenance.js

```diff
@@ -76,22 +76,19 @@ const updateAlert = async function(info, pgClient, discordClient, isComplete){
 		}
 	}
 
-
-	pgClient.query("SELECT messageID, channelID FROM alertMaintenance WHERE alertID = $1;", [info.instanceId])
-	.then(rows => {
-		for(let row of rows.rows){
```

Comment:
The for loop and `Promise.allSettled` should also work the same since the promises will be created without waiting for them to be resolved.